docs/contributor/02-30-keb-configuration.md

```diff
@@ -21,10 +21,6 @@ Kyma Environment Broker (KEB) binary allows you to override some configuration p
 | **APP_DATABASE_NAME** | Defines the database name. | `broker` |
 | **APP_DATABASE_SSLMODE** | Specifies the SSL Mode for PostgreSQL. See [all the possible values](https://www.postgresql.org/docs/9.1/libpq-ssl.html).  | `disable`|
 | **APP_DATABASE_SSLROOTCERT** | Specifies the location of CA cert of PostgreSQL. (Optional)  | None |
-| **APP_KYMA_VERSION** | Specifies the default Kyma version. | None |
```

Comment:
Although Kyma version param is not functional it is still present in multiple places in the code. Maybe we could add `deprecated` mark anywhere kymaVersion is specified until we remove it from code.